tests/asyncapi-2.0/AsyncAPI Version String/valid-major-minor.yaml

```diff
@@ -0,0 +1,16 @@
+asyncapi: 2.0
```

Comment:
Fixed. I guess I got confused because [here](https://github.com/asyncapi/asyncapi/blob/master/versions/2.0.0/asyncapi.md#fixed-fields) it says `shall be major.minor.patch` while [here](https://github.com/asyncapi/asyncapi/blob/master/versions/2.0.0/asyncapi.md#asyncapi-version-string) it actually says `must be major.minor.patch`.

Reply:
Good point. I'll fix it. Thanks!